modules/ding_groups/ding_groups.strongarm.inc

```diff
@@ -111,7 +151,7 @@ function ding_groups_strongarm() {
   $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
   $strongarm->api_version = 1;
   $strongarm->name = 'language_content_type_ding_group';
-  $strongarm->value = '0';
+  $strongarm->value = '2';
```

Comment:
What does this do?

Reply:
This shows that Multilingual support is enabled for this content type.
0 - Disabled
1 - Enabled
2 - Enabled with translation